pkgs/test_api/lib/src/backend/live_test_controller.dart

```diff
@@ -186,9 +193,15 @@ class LiveTestController {
     return liveTest.onComplete;
   }
 
+  /// Returns a future that completes when the test is complete.
+  ///
+  /// We also wait for the state to transition to Status.complete.
+  Future get onComplete =>
```

Comment:
nit: `Future<void>` (I realize the rest of the file doesn't use this style, but `Future<void>` didn't exist when it was written)

Reply:
Now added `<void>` in a bunch of places.